apps/docs-website/docs/vscode-support.md

```diff
@@ -38,10 +38,18 @@ For [Astro](https://astro.build/) projects, add the astro extension too:
 
 ## Avoid "source.organizeImports"
 
-Sheriff already handle imports sorting, so you have to disable thew VSCode automatic import sorting feature to avoid conflicts:
+Sheriff already handle imports sorting, so you have to disable VSCode automatic import sorting feature to avoid conflicts:
 
 ```JSONC title=".vscode/settings.json"
 "editor.codeActionsOnSave": {
     "source.organizeImports": false
 }
 ```
+
+If you wish, you can automatically fix some ESLint violations on save, including sorting imports:
+
+```JSONC title=".vscode/settings.json"
+"editor.codeActionsOnSave": {
+    "source.fixAll.eslint": "explicit"
```

Comment:
So, i know that a lot of other configs recommend this in the docs, but up until now i purposefully didn't include it.
The reason being, is that this is a generic ESLint usage recommendation which has nothing to do specifically with Sheriff, so i didn't want to clutter the docs with irrelevant informations, even though it could be useful for newbies approaching ESLint for the first time.
What do you think of my reasoning here?

Reply:
Yeah, but with import sorting I think it makes sense, as otherwise it's not a direct replacement to codeActionsOnSave.

Reply:
Agree